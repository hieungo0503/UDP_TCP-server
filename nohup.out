Traceback (most recent call last):
  File "/home/inter/Server_file/UDP_server_no_res.py", line 28, in <module>
    UDPServerSocket.bind((localIP, localPort))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: [Errno 98] Address already in use
Traceback (most recent call last):
  File "/home/inter/Server_file/UDP_server_no_res.py", line 28, in <module>
    UDPServerSocket.bind((localIP, localPort))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: [Errno 98] Address already in use
Traceback (most recent call last):
  File "/home/inter/Server_file/UDP_server_no_res.py", line 28, in <module>
    UDPServerSocket.bind((localIP, localPort))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: [Errno 98] Address already in use
