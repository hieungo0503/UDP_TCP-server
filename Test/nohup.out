python3: can't open file '/home/inter/Server_file/Test/UDP_server_no_res.py': [Errno 2] No such file or directory
